soh/soh/SohMenuBar.cpp

```diff
@@ -155,6 +155,17 @@ void DrawShipMenu() {
         }
 #if !defined(__SWITCH__) && !defined(__WIIU__)
         UIWidgets::Spacer(0);
+#if defined(__APPLE__) || defined(__WINDOWS__) || defined(__LINUX__)
+        if (ImGui::MenuItem("Open App Files Folder")) {
+#if defined(__WINDOWS__)
+            std::string filesPath = std::filesystem::current_path().string();
+#else
+            std::string filesPath = LUS::Context::GetInstance()->GetAppBundlePath();
```

Comment:
This points to the location of the executable, not the configuration files.
```suggestion
            std::string filesPath = LUS::Context::GetInstance()->GetAppDirectoryPath();
```

And it may be preferable to use this method for windows too, as it handles the portable/non-portable differences.

Reply:
That is good to know about non-Windows installs, I'll fix it when I get home. However, the intention is that there will be no difference between portable and non portable on the Windows side. The configuration files for the installer version will be in the application directory just like it is for the archive install. Besides which, the "non-portable" path is just hard coded to be `C:\Program Files\Soh`, which is an erroneous assumption anyway.

Reply:
Oh, wait, I'm probably still thinking of the app bundle function for where it's pointing to. I'll have to see where the other one is actually assigning for non portable. However, that still doesn't get into my plan for the config files to still be stored next to the executable for the installer version.

Reply:
The other thing to keep in mind about this whole process, at least from what I've been able to gather, the installer creation is run on top of the normal windows compilation. So, without creating an entirely new workflow just for the installer, we wouldn't be able to make an installer with its own non-portable defines in order to denote path differences like that.

Reply:
> However, that still doesn't get into my plan for the config files to still be stored next to the executable for the installer version.

This is irrespective of your plans for the installer. non-portable just means that the configuration dir can be separate of the executable dir. If non-portable is not enabled, `GetAppDirectoryPath` just returns `.` which is the current directory, like how you are already doing it. But if it is in non-portable mode, then `GetAppDirectoryPath` returns the location of the configuration files.

These methods are the exact methods being used to load the mods/ folder for example, so it makes sense to use it for the file explorer open flow.

Reply:
Ah, yes, now that I look at the other function, it does make sense to use that, as that will be modified should I decide to not differentiate between portable and non-portable for file location in the installer PR.

Reply:
As an addendum, though, I'll still have to do the modification to it for Windows since SDL_OpenURL requires an absolute path, and that function doesn't provide one for Windows.

Reply:
> As an addendum, though, I'll still have to do the modification to it for Windows since SDL_OpenURL requires an absolute path, and that function doesn't provide one for Windows.

Why only windows? Really should just convert to absolute path no matter what. There is a possibility that linux/mac could override the `SHIP_HOME` env var, so best to not assume those are going to be absolute.